lib/action_controller/serialization/context.rb

```diff
@@ -0,0 +1,18 @@
+module ActionController
+  module Serialization
+    class Context
+      include Rails.application.routes.url_helpers if Rails.application.present?
+
+      def self.default_url_options
+        ActionMailer::Base.default_url_options
+      end
```

Comment:
Oh sorry about `ActionMailer` this is me trying things by copy/pasting from SO in a hurry :( Will look in to how do it from the railtie!